.pyinstaller/run_sunpy_tests.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+import shutil
+
+import pytest
+
+# Skipping these tests that take check the name of the current module (which ends up starting with
+# sunpy_tests rather than sunpy).
+# asdf path issue with PyInstaller as well.
+
+if getattr(sys, 'frozen', False):
+    # Running in a bundle
+    bundle_dir = sys._MEIPASS
+
+    SKIP_TESTS = [
+        'test_saveframe',
+        'test_saveframe_arr',
+        'test_genericmap_basic',
+        'test_genericmap_mask',
+        'test_attr_metamagic',
+        'test_main_nonexisting_module',
+        'test_main_stdlib_module',
+        'test_origin',
+        'test_find_dependencies',
+        'test_missing_dependencies_by_extra',
+        'test_hgc_100',
+        'test_missing_dependencies_by_extra',
+        'test_basic',
+        'test_data_manager',
+        'test_file_tampered',
+        'test_download_cache',
+        'test_skip_all',
+        'test_override_file',
+        'test_same_file_id_different_module']
```

Comment:
Can any of these be removed now?

Reply:
From the script, these tests either use current directory's file name which is 'sunpy_tests' instead of 'sunpy' which resutls in an assertion error.
Or this is an issue with asdf, the asdf pathing seems to work fine when run without PyInstaller